### ISSUE TITLE
Stronger differ between levels of user activity on chat

### ISSUE THREAD

**Issue by SamanthaNguyen** [Brickimedia/brickimedia#448]
The color contrast between levels of user activity seems pretty weak and should be stronger so it's easier for users to tell whether or not users are active, somewhat inactive and completely idle. (right now, the idle/inactive color is `#F6F6F6` and the active color is `#EEEEEE`) 

Relying on color alone also is bad for users with color blindness or some form of color deficiency. Is there a better way we can do this? I was thinking icons but I don't know what kind of icon would be able to show each level of activity, and if it would be recognizable by users. It could also be confused with the user group icons

![image](https://cloud.githubusercontent.com/assets/4723983/12539749/f6b61caa-c2be-11e5-895d-ecb774386863.png)

**Comment by UltrasonicNXT** [comment]
There is a tooltip if you hover. I could make the colour change stronger though.

**Comment by codynguyen1116** [comment]
The tooltip is great, but it's a hidden affordance since there's not anything that conveys you can get information by hovering over it /besides/ color, which is still dangerous to use alone. Making the contrast higher is good, but there should something else that helps the user know how active those other people are. Suggestions (following how Skype does it)

* A clock for when someone is away
* A check for when someone is active
* An x when someone has been very idle

These would (probably) be recognizable to most users since Skype is used by many. We could use these icons from the WikiFont icon set to follow visual consistency with everything else also.

Thoughts and opinions on this?

**Comment by georgebarnick** [comment]
Clock makes sense to me maybe with their idle time in a short string next
to it like "5m" for 5 minutes or "1h" for 1 hour. Facebook Messenger does
that. An x for very idle doesn't make much sense to me and seeing an x by
someone's name would confuse me if I were new to the chat.

On Mon, Jan 25, 2016 at 8:25 PM Cody Nguyen <notifications@github.com>
wrote:

> The tooltip is great, but it's a hidden affordance since there's not
> anything that conveys you can get information by hovering over it /besides/
> color, which is still dangerous to use alone. Making the contrast higher is
> good, but there should something else that helps the user know how active
> those other people are. Suggestions (following how Skype does it)
>
>    - A clock for when someone is away
>    - A check for when someone is active
>    - An x when someone has been very idle
>
> These would (probably) be recognizable to most users since Skype is used
> by many. We could use these icons from the WikiFont icon set to follow
> visual consistency with everything else also.
>
> Thoughts and opinions on this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Brickimedia/brickimedia/issues/448#issuecomment-174757848>
> .
>
-- 
George Barnick
https://www.georgebarnick.photo

**Comment by codynguyen1116** [comment]
Your clock suggestion is good :+1: I've been thinking about the idle icon for a while after I made the comment actually and came up with ways we could replace the x (I didn't give much thought the first time):
 - grey crescent moon 
 - subtraction sign inside a circle 

The 3rd suggestion is to actually just have 2 icons, which is for showing who's active and away. That might just make it simpler for users to understand

**Comment by neoncitylights** [comment]
Brickimedia no longer uses GitHub as an issue tracker.
